### ISSUE TITLE
Control Character and character out of map

### ISSUE THREAD

**Issue by ztype** [signintech/gopdf#73]
Hi,
I use gopdf and got some questions,
The whole pdf file's characters will be invisible when I wrote a string contains Control Character like "\n" or "\r" into it. 
And other characters not contained in the `ttf` file will cause this problem too.

What should i do ?

appreciate your reply.

**Comment by oneplus1000** [comment]
I make some change in gopdf, by ignore glyph that not contained in .ttf file. 
Now I push to branch ["unknowglyph"](https://github.com/signintech/gopdf/tree/unknowglyph).
Let me know if you have any feedback :smiley:

**Comment by ztype** [comment]
thanks for your reply.

I have tried the `unknowglyph` branch and the problem seems still be there.

In master branch,when I write Control Characters or unkown Characters by using function `Text`,all the charaters be invisible but `Text` function has no error return.
In fact, I tried to solve this problem by myself after reading your code.
like that:
`for _, runeValue := range text {
		if runeValue <= 31 || runeValue == 127 {
			log.Printf("%v", runeValue)
			continue
		}
		err := pdf.Text(string(runeValue))
		if err != nil {
			log.Println(err)
		}
	}`
(I know that code will not work when the `runeValue` is other values.)
The `err` is `nil` when the problem happened,so I still had no idea about if this character is in the `ttf` or not.

So, I think could there be a function that I can use to know is the rune value in the `ttf` file or not before writing it? so I can do something instead,such as change another ttf file or set a default value to it :)

And,it will be better if I can get some detail information about the pdf object,such as the current font name,current font size,current page size,left margin etc. ,Just advice.

thanks again.
